Source/Controls.cs

```diff
@@ -84,13 +84,16 @@ public static void Consume()
 
 	private static string GetPromptLocation(string name)
 	{
+		var gamepadPure = Input.Controllers[0];
 		var gamepad = Input.Controllers[0].Gamepad;
 
 		if (!prompts.TryGetValue(gamepad, out var list))
 			prompts[gamepad] = list = new();
 
 		if (!list.TryGetValue(name, out var lookup))
-			list[name] = lookup = $"Controls/{GetControllerName(gamepad)}/{name}";
+		        list[name] = lookup = gamepadPure.Name == "Unknown"

```

Comment:
Instead of relying on the string name, can we use `gamepadPure.Connected` ?

Reply:
Yeah.